frontend-ssr/pages/submissions.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "../styles/variables";
+.page-submissions {
+  width: $width;
```

Comment:
Let's extract those 3 rules in a placeholder - what we're already doing with all the gradients. And then here and in the `Lesson.module.scss` we just inlclude it. This way we remove some duplicate styles.